src/components/shared/Modal.js

```diff
@@ -1,23 +1,20 @@
 // @flow
 
 import PropTypes from "prop-types";
-import React, { Component } from "react";
-import type { Children } from "react";
+import * as React from "react";
```

Comment:
This is required to make use of the `React.Node` or `React.Element` types. Alternatively, we could do something like

```js
import type { Node } from "react"
```

Reply:
I like this